resilience4j-circuitbreaker/build.gradle

```diff
@@ -1,5 +1,6 @@
 dependencies {
     api project(':resilience4j-core')
+    implementation 'org.springframework.boot:spring-boot-starter:2.5.5'
```

Comment:
Hi,

resilience4j-circuitbreaker must not have a dependency to spring-boot-starter.
The library is framework agnostic and should not introduce any framework-specific dependencies.